files/files.go

```diff
@@ -26,3 +26,13 @@ func CopyFile(src, dest string) error {
 	_, err = io.Copy(destination, source)
 	return err
 }
+
+func WriteFile(src io.ReadCloser, dest string) error {
```

Comment:
Can *probably* just use io.Reader for the input class, since it doesn't care that it's a closer or not